client/my-sites/themes/theme-showcase.scss

```diff
@@ -184,6 +184,116 @@ body.is-section-themes {
 	}
 }
 
+.is-logged-out {
+	.themes__header::before,
+	.themes__content::before,
```

Comment:
Looks like we might not need `.themes__content::before` 🤔

Reply:
`.themes__content::before` is used below to give the showcase a white background  🙂

```
	.themes__content::before {
		background-color: var(--studio-white);
	}
```

Reply:
Yes but it looks good without `.themes__content::before` on my side 
<img width="1069" alt="image" src="https://user-images.githubusercontent.com/13596067/223951482-d4dddd9a-8263-473f-a7dc-d0521c2868e4.png">

Reply:
Interesting, I tried it again and it works as the screenshot you attached. Trying again in the calypso.live build and it looks fine without `.themes__content::before` 🙂

Reply:
Let's wait for the translations 🙂